cli/index.ts

```diff
@@ -6,6 +6,15 @@ import * as chokidar from "chokidar";
 import * as protos from "@dataform/protos";
 import { init, compile, build, run, table, query, utils } from "@dataform/api";
 
+const addCompileYargs = (yargs: yargs.Argv) =>
+  yargs
+    .option("default-schema-override", {
```

Comment:
I don't think we need -override suffix, adds a lot of length, feel like it's fairly clear what this is going to do?